net/xl2tpd/Makefile

```diff
@@ -8,7 +8,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=xl2tpd
-PKG_VERSION:=devel-20151125
+PKG_VERSION:=devel-20160811
 PKG_RELEASE:=1
```

Comment:
It's `PKG_RELEASE` that should be increased.  The date in `PKG_VERSION` is of the xl2tpd commit packaged.